src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -515,18 +514,19 @@ def _test_flow_multi_modal(args, pf_client):
         from promptflow._sdk._tracing import _invoke_pf_svc
 
         pfs_port = _invoke_pf_svc()
+        serve_app_port = find_available_port()
         flow = generate_yaml_entry_without_recover(entry=args.flow)
         # flex flow without yaml file doesn't support /eval in chat window
         enable_internal_features = Configuration.get_instance().is_internal_features_enabled() or flow != args.flow
```

Comment:
please help change this to Configuration.get_instance().is_internal_features_enabled() **and** flow != args.flow, Thanks!